src/app/clusters/unit-localization-server/BUILD.gn

```diff
@@ -11,5 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-group("unit-localization-server") {
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+source_set("unit-localization-server") {
+  sources = [
+    "UnitLocalizationCluster.cpp",
```

Comment:
These should not be here until they are decoupled from codegen.

In particular these use `AttributeAccessInterfaceRegistry::Instance()` and those are only processed by ember. Even though compilation does work, I would prefer these to be kept in app-specific for now and we move them here when we convert the cluster.